pgctl/service.py

```diff
@@ -150,16 +165,36 @@ def ensure_exists(self):
         if not self.path.check(dir=True):
             raise NoSuchService("No such service: '%s'" % bestrelpath(str(self.path)))
 
-        # ensure symlink {service_dir}/supervise -> {scratch_dir}/supervise
-        # this will re-connect the service to its state descriptors if the symlinks have been deleted or moved
-        supervise_in_scratch = self.scratch_dir.join('supervise')
-        supervise_in_scratch.ensure_dir()
-        symlink_if_necessary(supervise_in_scratch, self.path.join('supervise'))
+        self._ensure_supervise_is_scratch('supervise')
         self.__exists = True
 
     def ensure_logs(self):
         self.ensure_exists()
-        self.path.ensure('log')
+        self.path.ensure_dir('logs')
+        self.path.join('logs').ensure('current')
+
+        self.path.ensure_dir('log')
```

Comment:
here's sort of a crazy idea...

what if we put the log service directories (not the log files themselves, though) under the scratch dir in the home directory? maybe we could even symlink into scratch to make it easier to find if you're looking for it.

i think the status quo on this branch might be somewhat confusing, since we're creating a whole bunch of new files but don't want users to modify them, which is confusing because for _most_ `run` files, we _do_ want users to modify them